WG/src/main/java/net/goldtreeservers/worldguardextraflags/flags/Flags.java

```diff
@@ -32,6 +32,8 @@
 	public final static SetFlag<String> CONSOLE_COMMAND_ON_EXIT = WorldGuardUtils.getCommunicator().getCustomSetFlag("console-command-on-exit", new CommandStringFlag(null));
 	
 	public final static DoubleFlag WALK_SPEED = new DoubleFlag("walk-speed");
+	public final static DoubleFlag FLY_SPEED = new DoubleFlag("fly-speed");
+	public final static DoubleFlag SPECTATE_SPEED = new DoubleFlag("spectate-speed");
```

Comment:
I think this is overkill, adds bloat and is really case specific

Reply:
Alright. Will be dropped in my next push.